src/components/repos/RepoList.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchRepos } from '../../actions/repoActions';
+import RepoDetail from './RepoDetail';
+
+const RepoList = () => {
+  const repos = useSelector(state => state.list);
+  const loading = useSelector(state => state.loading);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchRepos());
+  }, []);
```

Comment:
I don't think this useEffect is doing anything cause you aren't passing a username to the action